<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Jogo Jokenpo</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/jokenpo.png</normaloff>../img/jokenpo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background: black
</string>
  </property>
  <widget class="QLabel" name="lbl_nome">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>681</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat Black</family>
     <pointsize>26</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgba(255, 255, 255, 0);
color: rgb(85, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;JOGO JOKENPO&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_pedra">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/pedra.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_papel">
   <property name="geometry">
    <rect>
     <x>288</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/folha-de-papel.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_tesoura">
   <property name="geometry">
    <rect>
     <x>496</x>
     <y>160</y>
     <width>128</width>
     <height>128</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>128</width>
     <height>128</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../img/tesoura.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_pedra">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>310</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127);
color: rgb(255, 255, 255);
outiline:none;
border: none;</string>
   </property>
   <property name="text">
    <string>PEDRA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_papel">
   <property name="geometry">
    <rect>
     <x>302</x>
     <y>310</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127);
color: rgb(255, 255, 255);
outiline:none;
border: none;</string>
   </property>
   <property name="text">
    <string>PAPEL</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_tesoura">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>100</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 85, 127);
color: rgb(255, 255, 255);
outiline:none;
border: none;</string>
   </property>
   <property name="text">
    <string>TESOURA</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sorteio">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>490</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;COMPUTADOR:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sorteio_computador">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>488</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777203</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(255, 85, 127);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_sorteio_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>590</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_selecionado">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>410</y>
     <width>681</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Montserrat</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
